h/static/scripts/directive/test/annotation-share-dialog-test.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+var angular = require('angular');
+var util = require('./util');
+describe('annotationShareDialog', function () {
+  var element;
+
+  before(function () {
+    angular.module('app', [])
+      .directive('annotationShareDialog',
+        require('../annotation-share-dialog'));
+  });
+
+  beforeEach(function () {
+    angular.mock.module('app');
+  });
+
+  describe('vm.copyToClipboard()', function () {
+    var stub;
+
+    beforeEach(function () {
+      stub = sinon.stub(document, 'execCommand');
+      element = util.createDirective(document,
+        'annotationShareDialog',
+        {
+          group: {
+            name: 'Public',
+            type: 'public',
+            public: true
+          },
+          uri: 'fakeURI',
+          isPrivate: false
+        }
+      );
+    });
+
+    afterEach(function () {
+      stub.restore();
+    });
+
+    it('sets success class on successful copy to clipboard', function () {
+      var copyBtn = element.find('.annotation-share-dialog-link__btn');
```

Comment:
I suggest extracting these two lines into a helper function (eg. `getCopyBtn()`) which you can re-use in each test.